modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+=== Compatibility and support matrix
+
+Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table, features are marked with the following statuses:
+
+* *TP*: _Technology Preview_
+* *GA*: _General Availability_
+
+|===
+|*OpenShift GitOps* 6+|*Component Versions*|*OpenShift Versions*|*Support status*
+
+|*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*||
+|1.3.1|0.0.40|3.6.0|4.2.0|2.1.2|0.2.0|2.28.0|4.7-4.9|GA
+|1.3.0|0.0.40|3.6.0|4.2.0|2.1.2|0.2.0|2.28.0|4.7-4.9|GA
+|1.2.1|0.0.39|3.5.0|3.9.4|2.0.5|0.1.0|N/A|4.8|GA
+|1.2.0|0.0.38|3.5.0|3.9.4|2.0.5|0.1.0|N/A|4.8|GA
+|1.1.2|0.0.33|3.5.0|3.9.4|2.0.0|N/A|N/A|4.7|GA
+|1.1.1|0.0.32|3.5.0|3.9.4|2.0.0|N/A|N/A|4.7|GA
+|1.1.0|0.0.32|3.5.0|3.9.4|2.0.0|N/A|N/A|4.7|GA
```

Comment:
@DebarghoGhosh as Siamak mentioned we can drop the x.y.z release mentions in the table. So we can remove entries for 1.1.1, 1.1.2, 1.2.1, and 1.3.1